src/databricks/labs/ucx/install.py

```diff
@@ -285,14 +286,18 @@ def configure(self, default_config: WorkspaceConfig | None = None) -> WorkspaceC
         try:
             config = self.installation.load(WorkspaceConfig)
             self._compare_remote_local_versions()
-            if self._confirm_force_install():
-                return self._configure_new_installation(default_config)
-            self._apply_upgrades()
-            return config
+            if not self._confirm_force_install():
+                self._apply_upgrades()
+                return config
         except NotFound as err:
             logger.debug(f"Cannot find previous installation: {err}")
         except (PermissionDenied, SerdeError, ValueError, AttributeError):
             logger.warning(f"Existing installation at {self.installation.install_folder()} is corrupted. Skipping...")
+        except TimeoutError as err:
+            # API calls are wrapped with a retry mechanism for ConnectionError
+            # eventually raising a TimeoutError if tries take too long
+            if isinstance(err.__cause__, ConnectionError):  # raise TimeoutError(...) from ConnectionError(...)
+                logger.warning(f"Cannot connect with {self.workspace_client.config.host}: {err}")
```

Comment:
@nfx : What message would you like to have here? We can include something about a proxy, but that does not always have to be the reason the connection error is raised

Reply:
@JCZuurmond the best thing would a link to https://github.com/databrickslabs/ucx#network-connectivity-issues and expand documentation over there a bit.